07.ls_object/display.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class Display
+  def initialize(entries)
+    @entries = entries
+  end
+
+  def display_entries(long_format: false)
+    long_format ? display_long : display_short
+  end
+
+  private
+
+  def display_short
+    padding = max_length(:name) + 1
+    terminal_cols = `tput cols`.to_i
+    output_rows = @entries.size.ceildiv((terminal_cols / padding).floor)
+    output_rows.times do |row|
+      row.step(@entries.size - 1, output_rows) do |col|
+        print @entries[col].name.ljust(padding)
+      end
+      print "\n"
+    end
+  end
+
+  def display_long
+    puts "total #{@entries.sum(&:entry_blocks)}"
+    paddings = generate_row_width
+    @entries.each do |long_entry|
```

Comment:
```suggestion
    @entries.each do |entry|
```

entryそのものはlong/shortで違いはないはずです。

Reply:
修正しました。